### DISCUSSION TITLE
How to organize votefor 2 race

### DISCUSSION THREAD

**Discussion by kuriwaki** [kuriwaki/cvr_harvard-mit_scripts#139] [General]
The Arizona state house is a votefor 2 office, where a voter can vote for up to 2 candidates and the top 2 vote getters win. 

How do we code this in the database? 

We already have a `magnitude` variable, which seems to provide all the info we need.  But posting it here for discussion.

https://en.wikipedia.org/wiki/2020_Arizona_House_of_Representatives_election

**Comment by mreece13** [comment]
I think the magnitude variable is fine for now, unless you see a reason to change it. We are also using the magnitude variable for down-ballot races that are >1.
